src/main/java/org/fairdatatrain/trainhandler/utils/ValueFactoryUtils.java

```diff
@@ -0,0 +1,129 @@
+
+import java.time.OffsetDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Optional;
+
+public class ValueFactoryUtils {
+
+    private static final String PREFIX_SEP = ":";
+
+    private static final ValueFactory VF = SimpleValueFactory.getInstance();
+
+    public static IRI i(String iri) {
+        if (iri == null) {
+            return null;
+        }
+        return VF.createIRI(iri);
+    }
+
+    public static IRI i(Optional<String> optionalIri) {
+        if (optionalIri.isEmpty()) {
+            return null;
+        }
+        return i(optionalIri.get());
+    }
+
+    public static IRI i(String iri, Model model) {
+        if (iri != null) {
+            // URI: ':title'
+            if (iri.startsWith(PREFIX_SEP)) {
+                final Optional<Namespace> optionalNamespace = model.getNamespace("");
+                final String prefix = optionalNamespace.get().getName();
+                return i(prefix + iri.substring(1));
+            }
+
+            // URI: 'rda:title'
+            final String[] splitted = iri.split(PREFIX_SEP);
+            if (splitted.length == 2 && splitted[1].charAt(0) != '/') {
+                final Optional<Namespace> optionalNamespace = model.getNamespace(splitted[0]);
+                final String prefix = optionalNamespace.get().getName();
+                return i(prefix + splitted[1]);
+            }
+
+            // URI: 'http://schema.org/person#title'
+            if (iri.contains("://")) {
+                return i(iri);
+            }
+
+        }
+        return null;
+    }
+
+    public static IRI i(Value value) {
+        if (value == null) {
+            return null;
+        }
+        return i(value.stringValue());
+    }
+
+    public static Literal l(String literal) {
+        return VF.createLiteral(literal);
+    }
+
+    public static Literal l(Optional<String> oLiteral) {
+        if (oLiteral.isEmpty()) {
+            return null;
+        }
+        return l(oLiteral.get());
+    }
+
+    public static Literal l(float literal) {
+        return VF.createLiteral(literal);
+    }
+
+    public static Literal l(OffsetDateTime literal) {
+        if (literal == null) {
+            return null;
+        }
+        return VF.createLiteral(
+                DateTimeFormatter.ISO_OFFSET_DATE_TIME.format(literal),
+                XSD.DATETIME
+        );
+    }
+
+    public static Literal l(Value value) {
+        if (value == null) {
+            return null;
+        }
+        return l(value.stringValue());
+    }
+
+    public static Statement s(Resource subject, IRI predicate, Value object) {
+        return VF.createStatement(subject, predicate, object);
+    }
+
+    public static Statement s(Resource subject, IRI predicate, Value object, Resource context) {
+        return VF.createStatement(subject, predicate, object, context);
+    }
+
+    public static Resource bn() {
+        return VF.createBNode();
+    }
```

Comment:
Can be replaced by [`Values.bnode`](https://rdf4j.org/javadoc/latest/org/eclipse/rdf4j/model/util/Values.html#bnode())